src/layout/barGrid.ts

```diff
@@ -441,14 +466,40 @@ function retrieveColumnLayout(
 }
 export {retrieveColumnLayout};
 
+function orderLayoutData(layoutInfo: layoutInfo) {
+    const groupOrder = layoutInfo.groupOrder;
+    let layoutDataList = layoutInfo.layoutDataList;
+    let columnOffsetList = layoutInfo.columnOffsetList;
+    columnOffsetList.sort(function (a, b) {
+        return a - b;
+    });
```

Comment:
Since the love of anonymous function is shown here, I would suggest to make this anonymous function really compact:

columnOffsetList.sort((a, b) => a-b )

Reply:
I will submit it again later to fix the problem